README.md

```diff
@@ -106,6 +106,8 @@ Please ensure that you have at least the **minimum** recommended versions
 
 We use [jest](https://github.com/facebook/jest) as our test framework and [puppeteer](https://github.com/GoogleChrome/puppeteer) for browser automation in our end-to-end UI tests.
 
+> We have a known issue with running E2E on WSL in Windows 10. Issue posted [here](https://github.com/microsoft/accessibility-insights-web/issues/853).
```

Comment:
should we rephrase this to something like???:
`There is a know issue`
As mention before... this is not necessarily an issue with our code or deps, this is a known limitation of WSL (out of the box, power users may be able to get graphical applications running...)

Reply:
+1. I don't think "known issue" is a fair description. I think it would be more reasonable to instead say something like "this project's end to end tests require the ability to run a non-headless chromium process. Because of this, they are incompatible with non-graphical development environments (notably, a default WSL environment on Windows)"

Reply:
I will take that line @dbjorge . Thanks.